source-code/sdk-js/src/helpers/ast.test.ts

```diff
@@ -0,0 +1,215 @@
+import { describe, expect, test } from "vitest"
+import { parse } from "acorn"
+import { wrapVariableDeclaration, insertAst } from "./ast.js"
+// @ts-ignore
+import { generate } from "astring"
```

Comment:
See here: https://github.com/davidbonnet/astring/pull/668
Please use `patch-package` to patch it instead of ignoring it.

Reply:
Will fix